editors/webstorm/README.md

```diff
@@ -43,3 +43,16 @@ Go to *File | Settings | Tools | File Watchers* for Windows and Linux or *WebSto
 * **Immediate file synchronization**: Uncheck to reformat on Save only (otherwise code will jump around while you type).
 
 ![Example](./prettier-file-wacther.png)
+
+### Multiple file types  ###
+
+If you woud like to use one watcher for multiple file types.
```

Comment:
Fragment. You need a comma at the end of this sentence.

Reply:
There's a period there

Reply:
I think @azz meant this:

> If you would like to use one watcher for multiple file types, or if you would like more control, make these changes to the settings above:

Reply:
`woud` -> `would`